### ISSUE TITLE
Feature request: add / edit / delete tags for existing snapshots

### ISSUE THREAD

**Issue by lcd047** [gilbertchen/duplicacy#112]
Currently tags can be created only by the `backup` command, and then they remain immutable for the entire life of the repository (or at least I haven't found any way to change them).  It might be useful to be able to add, edit (i.e. rename, I suppose), or delete tags for existing snapshots.

**Comment by gilbertchen** [comment]
Right, a user may forget to provide a tag when running the backup command, so having a command to change/add the tag should be necessary.

I'll add this to my to-do list.

**Comment by ddzero2c** [comment]
Hi Gilbert
Could I restore by tag without list all revision and grep ?
